<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="action_server_name" default="long_dummy_action_server" description="Name of the action server"/>
    <arg name="action_duration" default="60" description="Duration of dummy action"/>

    <node pkg="long_dummy_action" exec="long_dummy_action" name="long_dummy_action" output="screen" emulate_tty="true">
        <param name="action_server_name" value="$(var action_server_name)"/>
        <param name="action_duration" value="$(var action_duration)"/>
    </node>
</launch>